src/App.tsx

```diff
@@ -1,13 +1,47 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import cn from 'classnames';
+
+const getFileteredTodos = (todos: Todo[], filter: Filter): Todo[] => {
+  return todos.filter(todo => {
+    if (filter === Filter.Completed) {
+      return todo.completed;
+    }
+
+    if (filter === Filter.Active) {
+      return !todo.completed;
+    }
+
+    return true;
+  });
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filterOption, setFilterOption] = useState<Filter>(Filter.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodosFromServer)
+      .catch(() => setError('Unable to load todos'))
+      .finally(() =>
+        setTimeout(() => {
+          setError('');
+        }, 3000),
+      );
+  }, []);
+
+  const uncompletedTodos = todosFromServer.filter(
```

Comment:
The name is not reflect the sense of variable because you use it as quantity but name tells us that it's more like a collection of completed todos.